ui/components/app/collectibles-items/index.scss

```diff
@@ -1,40 +1,47 @@
 .collectibles-items {
   &__image__wrapper {
-    border-radius: 0.375rem;
+    border-radius: 4px;
     width: 100%;
     display: flex;
     justify-content: center;
-    padding: 0.125rem;
+    cursor: pointer;
   }
 
   &__image {
-    border-radius: 0.625rem;
+    border-radius: 4px;
     width: 100%;
     height: 100%;
   }
 
   &__link {
     a {
-      padding: 0.25rem;
+      padding: 4px;
```

Comment:
I'm interested in hearing your perspective on `px` vs `rem`'s. Any reason to use one over the other?

Reply:
I prefer pixels for bounding box values(margin, padding, border) because:
- There aren't really any accessibility benefits from using rems for space. Generally speaking we want to keep the spacing between elements the same no matter what the base font-size is set to.
- Designers will tend to use pixel values in figma so it's one less calculation you have to do in your head.

I suppose one benefit to using rems is that round numbers and .5 numbers adhere to the 8px grid.

Reply:
Gotcha that makes sense! Thanks.